src/validator.test.ts

```diff
@@ -109,29 +109,22 @@ describe('validator', () => {
 
   describe('verifyAMTVersion', () => {
     test('should throw an exception if version is less than 7', async () => {
```

Comment:
This test is a perfect candidate for a table/data driven test to handle all scenarios.

Reply:
done